tests/unit/test_visualization.py

```diff
@@ -276,6 +280,191 @@ def test_get_column_plot_no_data():
         get_column_plot(None, None, 'values')
 
 
+@patch('sdmetrics.visualization.px.histogram')
+def test__generate_column_bar_plot(mock_histogram):
+    """Test ``_generate_column_bar_plot`` functionality"""
+    # Setup
+    real_data = pd.Series([1, 2, 2, 3, 5])
+    synthetic_data = pd.Series([2, 2, 3, 4, 5])
+
+    # Run
+    _generate_column_bar_plot(real_data, synthetic_data)
+
+    # Assert
+    expected_data = pd.DataFrame(
+        pd.concat([real_data, synthetic_data], axis=0, ignore_index=True).astype('float64')
+    )
+    expected_parameters = {
+        'x': 'values',
+        'color': 'Data',
+        'barmode': 'group',
+        'color_discrete_sequence': ['#000036', '#01E0C9'],
+        'pattern_shape': 'Data',
+        'pattern_shape_sequence': ['', '/'],
+        'histnorm': 'probability density',
+    }
+    pd.testing.assert_frame_equal(expected_data, mock_histogram.call_args[0][0])
+    assert expected_parameters == mock_histogram.call_args[1]
+    mock_histogram.assert_called_once()
+
+
+@patch('sdmetrics.visualization.ff.create_distplot')
+def test__generate_column_distplot(mock_distplot):
+    """Test ``_generate_column_distplot`` functionality"""
+    # Setup
+    real_data = pd.DataFrame({'values': [1, 2, 2, 3, 5]})
+    synthetic_data = pd.DataFrame({'values': [2, 2, 3, 4, 5]})
+
+    # Run
+    _generate_column_distplot(real_data, synthetic_data)
+
+    # Assert
+    expected_data = []
+    expected_data.append(real_data['values'])
+    expected_data.append(synthetic_data['values'])
+    expected_data == mock_distplot.call_args[0][0]
+
+    ['Real', 'Synthetic'] == mock_distplot.call_args[0][1]
+
+    expected_colors = [PlotConfig.DATACEBO_DARK, PlotConfig.DATACEBO_GREEN]
+    expected_parameters = {
+        'show_hist': False,
+        'show_rug': False,
+        'colors': expected_colors,
+    }
+    assert expected_parameters == mock_distplot.call_args[1]
+    mock_distplot.assert_called_once()
```

Comment:
minor: I think for these you can do 
```python
mock_distplot.assert_called_once_with(show_hist=False, show_rug=False ...)
```
The only time I think you'd need to pull out the call args itself is to evaluate dataframes

Reply:
Updated so I can use `assert_called_once_with`. I had to update it to use `ANY` from `unittest.mocks`